File: java_backup/java.jimut/constructors/factorial.java
public class factorial
{
    int a;
    factorial()
    {
        a=0;
    }
    public void input(int m)
    {
        a=m;
    }
    public void display()
    {
        int i,f=1;
        for(i=1;i<=a;i++)
        {
            f=f*i;
        }
        System.out.println("The factorial of "+a+" is ::"+f);
    }
}
            

File: python/python_new/Python 3/factorial.py
num=int(input("Enter a number: "))
fac=1
if num<0:
 print("Negative number")
elif num==0:
 print("Factorial of 0 is 1")
else:
 for i in range(1,num+1):
  fac=fac*i
 print("Factorial of ",num," is ",fac)
